Failing codestyle checks due to  latest Python/pylint

Running tests locally with latest Python 3 makes latest pylint (at present 3.3.x) to be used which breaks codestyle checks. When running with Python 3.8, pylint 3.2.7 is used. "Too many..." was added as a part of pylint 3.3.0.

### Log

Running pylint for /home/eannjon/workarea/ecchronos_github/ecchronos/ecchronos-binary/src/bin

--------------------------------------------------------------------
Your code has been rated at 10.00/10 (previous run: 10.00/10, +0.00)

Running pylint for /home/eannjon/workarea/ecchronos_github/ecchronos/ecchronos-binary/src/pylib/ecchronoslib
************* Module ecchronoslib.rest
/home/eannjon/workarea/ecchronos_github/ecchronos/ecchronos-binary/src/pylib/ecchronoslib/rest.py:242:4: R0917: Too many positional arguments (6/5) (too-many-positional-arguments)

------------------------------------------------------------------
Your code has been rated at 9.98/10 (previous run: 9.98/10, +0.00)

[ERROR] Command execution failed.
org.apache.commons.exec.ExecuteException: Process exited with an error: 1 (Exit value: 1)


### Workaround/fix?

$ git diff
diff --git a/ecchronos-binary/src/pylib/ecchronoslib/rest.py b/ecchronos-binary/src/pylib/ecchronoslib/rest.py
index 99de38d6..bae52559 100644
--- a/ecchronos-binary/src/pylib/ecchronoslib/rest.py
+++ b/ecchronos-binary/src/pylib/ecchronoslib/rest.py
@@ -239,7 +239,7 @@ class V2RepairSchedulerRequest(RestRequest):
             result = result.transform_with_data(new_data=[Repair(x) for x in result.data])
         return result
 
-    def get_repair_info(self, keyspace=None, table=None, since=None,  # pylint: disable=too-many-arguments
+    def get_repair_info(self, keyspace=None, table=None, since=None,  # pylint: disable=too-many-arguments,too-many-positional-arguments
                         duration=None, local=False):
         request_url = V2RepairSchedulerRequest.repair_info_url
         if keyspace:


Should probably add the new disable and look into stepping the actions to using a newer Python 3 than 3.8.

--- a/.github/workflows/actions.yml
+++ b/.github/workflows/actions.yml
@@ -109,10 +109,10 @@ jobs:
           java-version: ${{ matrix.java_version }}
           distribution: 'temurin'
       - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
-      - name: Set up Python 3.8
+      - name: Set up Python 3
         uses: actions/setup-python@0a5c61591373683505ea898e09a3ea4f39ef2b9c # v5.0.0
         with:
-          python-version: 3.8
+          python-version: 3
       - name: install dependencies
         run: mvn install -DskipTests=true
       - run: mvn $TEST_SUITE -B

--- a/ecchronos-binary/src/pylib/ecchronoslib/rest.py
+++ b/ecchronos-binary/src/pylib/ecchronoslib/rest.py
@@ -239,7 +239,7 @@ class V2RepairSchedulerRequest(RestRequest):
             result = result.transform_with_data(new_data=[Repair(x) for x in result.data])
         return result
 
-    def get_repair_info(self, keyspace=None, table=None, since=None,  # pylint: disable=too-many-arguments
+    def get_repair_info(self, keyspace=None, table=None, since=None,  # pylint: disable=too-many-arguments, too-many-positional-arguments
                         duration=None, local=False):
         request_url = V2RepairSchedulerRequest.repair_info_url
         if keyspace:

